### ISSUE TITLE
Guard Overpowered

### ISSUE THREAD

**Issue by Slamm21** [EdenServer/community#5901]
### Checklist

- [ ] I have searched for existing issues for issues like this one. The issue has not been posted.
- [ ] I have provided reproducible steps. (No "as the title says" posts please. ***Please try to state facts only, and avoid assumptions/opinions.***)
- [ ] I have provided my client version in the details. (type /ver into your game window)
- [ ] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details
Guard should reduce pdif by 1 on the first hit if it procs. This can mean, if PDIF is low enough already a hit for 0.
It will only apply on the first hit of a multi hit

On Eden I can be fighting a mnk in dynamis where I know I have capped ratio and it will guard for 0. This should be impossible
https://www.bg-wiki.com/ffxi/Guard_Skill
https://ffxiclopedia.fandom.com/wiki/Guarding_Skill

**Comment by Steam-arch** [comment]
Whenever this gets checked, Guard rate of Monsters could be checked aswell because it seems like it does scale off high with high level Monsters.

Feels like SE would have implemented either a guard cap to Monsters or given them a base guard rate due to balancing, because having 10% base counter and a guard which scales ends up makes high level MNK mobs really tough.

Or edens guard calculations might be off in general.

I did a era video comparison once with an old and shortish Sabotender bailarina Clip and The guard rate was similar to the base counter rate of 10%, edens bailarinas Guard rate was more than double that. Seems to Be an occurring patterin as Ix'mnk on Eden has crazy defence capabilities aswell with counter + high guard rate.

Defenitely things abit fishy with Guard

**Comment by Slamm21** [comment]
> Whenever this gets checked, Guard rate of Monsters could be checked aswell because it seems like it does scale off high with high level Monsters.
> 
> Feels like SE would have implemented either a guard cap to Monsters or given them a base guard rate due to balancing, because having 10% base counter and a guard which scales ends up makes high level MNK mobs really tough.
> 
> Or edens guard calculations might be off in general.
> 
> I did a era video comparison once with an old and shortish Sabotender bailarina (lvl 80-82) Clip and The guard rate was similar to the base counter rate of 10%, edens bailarinas Guard rate was atleast double that. Seems to Be an occurring patterin as Ix'mnk on Eden has crazy defence capabilities aswell with counter + high guard rate.
> 
> Defenitely things abit fishy with Guard
For players vs mobs, we know that both guard and parry were floored at 5% on anything worth a damn, but you could see them proc a fair bit on too weak mobs

It's not a job trait like Double attack so it is unlikely to be a base rate for all mobs. It's a skill like evasion.
That being said, a level 95 MNK mob would have 389 skill and proc rate should be their skill compared to their enemey (your) level

If we just assume guard skill - combat skill, combat skill would be 276+16+7 with torque and merits = 299
389-299 = 90, does this mean 9% which considering videos seem to cap at 10% isn't a bad coincidence if it is one.
no merit or torque = 11.3%

If this was the case, you could proc over 5% on mobs in the low 60s at 75, but otherwise you would remain floored at 5%. Guard and Parry were never meant to proc a lot as they come last in the series of checks

Not saying the math there is correct, but it could be or there could be another number in the equation. For Attack, we use 8 base and STR/2 or STR*0.75. Evasion has AGI/2 but I think it is unlikely guard uses stat/2 as no testing has found a correlation and it would make it too powerful. If anything I'd guess at a tiny starting number just like how attack has a base of 8.

**Comment by Steam-arch** [comment]
Yeah never seen an actual formula for it. Feels like there might be something extra added to edens formula with even Ix MNK (lvl83) seemingly breaking +10% with ease and thats a ~40 skills difference pre merits/Gear.

**Comment by TracentEden** [comment]
Yeah, I have been looking into guard for LSB (as I made a PR for adding guard to WS there). The formula on LSB is 11 years old and has no justification in the commit or PR. Thus I am not sure of the original source. 

Relatedly, as far as the mob types that should have guard, Siknoz has recently done some retail captures ([video](https://youtu.be/3_prTzTJ95U) and [files](https://1drv.ms/u/c/87e665e10555c452/Eb9WVxIq9ZZNq4zcKc1HYzQBgpw2X-qHD7N8Qv7yRxzDYQ?e=WyhTrs)) and found the following (maybe Eden already this info but will add here just for reference). Also most of these tests showed floored guard rates of 5-6% (thus more captures probably needed to validate current formula).

**Have Guard:**
Orcish Champion
Yagudo Initiate
Mandragora
Fomor Monk
Korrigan
Arrapago Apkallu
Sabotender Corrido
Molech
Hilltroll Puppetmaster
Bugbear Deathman
Eo'aern (mnk)

**No Guard:**
Sea Monks (actually war)
Kindred Monk (dyna xarc, maybe using axe weapon?)
Ancient Goobbue (prob war)
Byakko
Genbu

**Comment by Slamm21** [comment]
Kind of related, when you do the morrigan legs 35 path in SSR 2nd floor, those mnk fomors are OP as hell.
You go from squishy weak mob with the other fomors to need to seigan third eye immediately before you die + guard making them very tanky.
